### ISSUE TITLE
Web Panels in Brave

### ISSUE THREAD

**Issue by jsandiego** [brave/brave-browser#4806]
I see that https://github.com/brave/browser-laptop/issues/1873 was closed in favour of this one.
However, they are not the same.

https://github.com/brave/browser-laptop/issues/1873 is about side panel, while this one is about vertical tabs... I don't see why they should be merged in the same ticket.

_Originally posted by @danielfranca in https://github.com/brave/brave-browser/issues/464#issuecomment-449390905_

Creating a new issue because the previous issue got merged into a vertical tab issue, which is not the same.

Would like the Web Panel feature that Vivaldi browser has in Brave.  This is the main reason why I have not switched over to Brave on my work laptop.  Being able to quickly open up my Web Panel of my email, job status pages, etc is very important.

https://help.vivaldi.com/article/web-panels/

**Comment by devamin** [comment]
Please me too the same thing. we would like to add web panel to Brave it's super good feature.

**Comment by mlbrooks** [comment]
Web panels are the one feature that keep me using Vivaldi instead of Brave. I love everything else about Brave and wish I could switch, but I can't leave web panels behind.

**Comment by 1oglop1** [comment]
I'd love any kind of side panel, it could even support custom extensions which would be great.
I cannot imagine web browser on desktop (yes, many people still use desktop versions) without side panel.
Especially considering that resolutions are getting wider so adding anything to the top or bottom not very ergonomic.

**Comment by Alexander-Carpenter** [comment]
Any movement on this? Brave would be my preferred browser if they had anything similar.

**Comment by rolltidehero** [comment]
> Any movement on this? Brave would be my preferred browser if they had anything similar.

Agreed. This is the only reason that I still use Vivaldi over Brave. 
Well, the main reason. I also love the tiled tabs. 
But I love me some Brave!

**Comment by jordanengstrom** [comment]
I'd be interested in this. I've been playing with the Arc browser lately, and it offers split pane functionality similar to Vivaldi. Arc calls this feature "Split View".

**Comment by zeusshomaly** [comment]
@simonhong found another sidebar web panel issue you might need to close (it remains opened after 5 years). :) especially by working to add it as a flag, so thank you :)
